util/printchplenv

```diff
@@ -132,6 +128,32 @@ def print_mode(mode='list'):
         print_mode('runtime')
         stdout.write('CHPL_MAKE_LAUNCHER_SUBDIR=')
         print_mode('launcher')
+
+        third_party_pkgs = chpl_home+'/util/chplenv/third-party-pkgs'
+        all_3p_pkgs=subprocess.Popen([third_party_pkgs],
+                                     stdout=subprocess.PIPE).communicate()[0].strip()
+
+        link_args_3p = ''
```

Comment:
Since you convert this to a list later anyway, why not make it a list here? Then you could call link_args_3p.append(...) instead of using string concatenation.